### PR TITLE
Closing right-click menu on mouse down

### LINKED ISSUE #75: Right-Click Menu Stays Open When Moving Camera
## Summary
The right-click menu stays open when moving the camera. This can cause a selection and highlighting of options in the dropdown menus.

**Proposed Solution**: When having an open right-click menu, upon moving the camera with the right mouse button, close the menu again.

right-click-wrong

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
Fixes #75.

Closes the right-click menu on right mouse button down.

right-click-right

### CODE REVIEW

**Comment by Interrupt** [comment]
PRs with gifs are the best PRs!

File: DelvEdit/src/com/interrupt/dungeoneer/editor/ui/EditorUi.java [hunk 1]
```diff
@@ -611,4 +611,10 @@ public void touchUp(int x, int y, int pointer, int button) {
             }
         }
     }
+
+    public void touchDown(int x, int y, int pointer, int button) {
```
Comment (PythooonUser) [inline DelvEdit/src/com/interrupt/dungeoneer/editor/ui/EditorUi.java:615]: Only the `button` parameter is used, but the method signature is aligned with the counterpart `touchUp`.

Comment (joshuaskelly) [reply DelvEdit/src/com/interrupt/dungeoneer/editor/ui/EditorUi.java:615]: It is matching the interface of `InputProcessor` and I think this is perfectly fine to do.

**Review by joshuaskelly — APPROVED** [review]
Thanks for fixing this annoyance! 🤘

**Pull request merged**